common/taskScheduler.go

```diff
@@ -0,0 +1,14 @@
+package common
+
+import (
+	"github.com/fiware/VCVerifier/logging"
+	"github.com/procyon-projects/chrono"
+	"time"
+)
+
+func Schedule(task chrono.Task) {
+	_, err := chrono.NewDefaultTaskScheduler().ScheduleAtFixedRate(task, time.Duration(30)*time.Second)
```

Comment:
I am not familiar with the chrono lib, but does the scheduler need some termination?

Reply:
Good question. It probably does, but it was already like that before, I've just moved it to another place